services/storage/series_cursor.go

```diff
@@ -122,14 +130,87 @@ CLEANUP:
 func (c *indexSeriesCursor) Close() {
 	if !c.eof {
 		c.eof = true
-		if c.sitr != nil {
-			c.sitr.Close()
-			c.sitr = nil
+		if c.sqry != nil {
+			c.sqry.Close()
+			c.sqry = nil
 		}
 	}
 }
 
 // Next returns the next series
+//func (c *indexSeriesCursor) Next() *seriesRow {
```

Comment:
Remove?

Reply:
Good catch – I meant to strip that out 👍